migrations/2023-11-15-200123_community_aggregates_add_local_subscribers/up.sql

```diff
@@ -0,0 +1,57 @@
+-- Couldn't find a way to put local_subscribers right after subscribers
+ALTER TABLE community_aggregates
+    ADD COLUMN local_subscribers int8 NOT NULL DEFAULT 0;
+
+-- update initial value
+UPDATE
+    community_aggregates
+SET
+    local_subscribers = (
+        SELECT
+            COUNT(*)
+        FROM
+            community_follower
+        WHERE
+            community_follower.community_id = community_aggregates.community_id
+            AND community_follower.person_id IN (
+                SELECT
+                    id
+                FROM
+                    person
+                WHERE
+                    local));
+
+CREATE OR REPLACE FUNCTION community_aggregates_subscriber_count ()
```

Comment:
Rather than replace the existing function, I'd create a new trigger that only runs when `local = true`, and a new function for that column specifically.

Reply:
Check the `site_aggregates_community_insert` trigger for an example.